setup/generate_dataset_nflx.py

```diff
@@ -0,0 +1,144 @@
+"""
+    This file will featurize the nflx and saved the cached feature pandas dataframe
+    That dataframe can be loaded for downstream transformations, such as selection of subfields which is done below and saved in a sample_data.csv
+"""
+import os
+import sys ; sys.path.append('..')
+
+
+from tqdm.auto import tqdm
+import numpy as np
+
+
+import EDGAR
+
+
+data_dir = os.path.join('..', 'data')
+
+metadata = EDGAR.metadata(data_dir=data_dir)
+loader = EDGAR.downloader(data_dir=data_dir);
+parser = EDGAR.parser(data_dir=data_dir)
+
+
+
+
+# List of companies to process
+#tikrs = open(os.path.join(loader.path, '..', 'tickers.txt')).read().strip()
+#tikrs = [i.split(',')[0].lower() for i in tikrs.split('\n')]
+
+tikrs = ["nflx"]
+
+trainset = []
+
+
+for tikr in tikrs:
+    metadata.load_tikr_metadata(tikr)
+    annotated_docs = parser.get_annotated_submissions(tikr)
+    #annotated_docs = [annotated_docs[0]]
+    for doc in tqdm(annotated_docs, desc=f"Processing {tikr}", leave=False):
+        fname = metadata.get_10q_name(tikr, doc)
+        # Try load cached, otherwise regenerate new file
+        features = parser.featurize_file(tikr, doc, fname,force = True) 
```

Comment:
Looks good -> can you add args.force into this script as well (from generate_dataset.py) and the argparser lines.

We want force to be False by default, unless really necessary.